Readme.md

```diff
@@ -3,21 +3,79 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
+# Grid View for ASP.NET Web Forms - How to hide a grid column on the client
+<!-- run online -->
+**[[Run Online]](https://codecentral.devexpress.com/e4580/)**
+<!-- run online end -->
+
+This example demonstrates how to change the visibility of the [Grid View](https://docs.devexpress.com/AspNet/5823/components/grid-view)'s [columns](https://docs.devexpress.com/AspNet/3691/components/grid-view/concepts/data-representation-basics/columns) on the client without making a round-trip to the server. Check or uncheck the **Unit Price Column** check box to show or hide the **Unit Price** column.
+
+![Hide a grid column on the client](hide-a-column.png)
+
+## Overview
+
+Follow the steps below to change a [column](https://docs.devexpress.com/AspNet/3691/components/grid-view/concepts/data-representation-basics/columns)'s visibility on the client.
+
+### 1 Apply a CSS class to all cells of a column
+
+A column can include the following cells:
+
+* [Column Header](https://docs.devexpress.com/AspNet/3669/components/grid-view/visual-elements/column-header)
+* [Edit Cell](https://docs.devexpress.com/AspNet/3680/components/grid-view/visual-elements/edit-form)
+* [Data Cell](https://docs.devexpress.com/AspNet/3670/components/grid-view/visual-elements/data-cell)
+* [Filter Row Cell](https://docs.devexpress.com/AspNet/3684/components/grid-view/visual-elements/filter-row)
+* [Footer Cell](https://docs.devexpress.com/AspNet/3683/components/grid-view/visual-elements/footer-cell)
+* [Group Footer Cell](https://docs.devexpress.com/AspNet/3815/components/grid-view/visual-elements/group-footer)
+
+Apply your CSS class to the [CssClass](https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.webcontrols.style.cssclass?view=netframework-4.8#System_Web_UI_WebControls_Style_CssClass) property of every cell in a column:
+
+```aspx
+<dx:ASPxGridView ID="gvProducts" runat="server" AutoGenerateColumns="False" DataSourceID="dsProducts" KeyFieldName="ProductID">
+    <Columns>
+        <!-- ... -->
+        <dx:GridViewDataTextColumn FieldName="UnitPrice" VisibleIndex="4">
+            <HeaderStyle CssClass="unitPriceColumn" />
+            <EditCellStyle CssClass="unitPriceColumn" />
+            <CellStyle CssClass="unitPriceColumn" />
+            <FilterCellStyle CssClass="unitPriceColumn" />
+            <FooterCellStyle CssClass="unitPriceColumn" />
+            <GroupFooterCellStyle CssClass="unitPriceColumn" />
+        </dx:GridViewDataTextColumn>
+    </Columns>
+</dx:ASPxGridView>
+```
+
+### 2 Specify the display property of the CSS class
+
+The CSS class's [display](http://www.w3schools.com/cssref/pr_class_display.asp) property specifies the display behavior of an element. Set this property to `none` to hide the column or set it to `table-cell` to show the column.
```

Comment:
I'd like to avoid the plural possessive noun if possible. Can we say the following instead?

```suggestion
The [display](http://www.w3schools.com/cssref/pr_class_display.asp) property of the CSS class specifies the display behavior of an element. Set this property to `none` to hide the column or set it to `table-cell` to show the column.
```